cni-plugin/Dockerfile

```diff
@@ -14,9 +14,17 @@ COPY controller controller
 COPY cni-plugin cni-plugin
 RUN CGO_ENABLED=0 GOOS=linux go build -o /go/bin/linkerd-cni -v -mod=readonly ./cni-plugin/
 
-FROM gcr.io/linkerd-io/base:2020-06-08.01
+FROM debian:buster-20200514-slim
 WORKDIR /linkerd
-RUN curl -kL -o $(which jq) https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    iptables \
+    jq && \
```

Comment:
Do we not need `curl`, `dnsutils`, `iptables`, `nghttp2` here?

Reply:
from @alpeb, `cni-plugin` only need `iptables`

Reply:
👍

Reply:
Those are only debugging tools that were provided to the Debug container for convenience, but anyways let me give CNI a test to make sure everything is working as expected...

Reply:
I confirmed CNI is working as expected :+1:

Reply:
@alpeb Thanks 🙂